src/main/java/io/github/oliviercailloux/y2018/apartments/apartment/ApartmentFactory.java

```diff
@@ -60,12 +60,17 @@ public static Apartment generateRandomApartment() {
 	/**
 	 * This function aims to generate a new apartment with random characteristics.
 	 *
+	 * @param realAddress - True indicates that the address is necessarily real (may
+	 *                    have an {@link InvalidObjectException}) <br>
```

Comment:
`InvalidObjectException` n’étant pas un nom limpide, il est particulièrement utile d’indiquer en `@throws` les circonstances dans lesquelles cette exception surviendrait (je ne vois pas ce qui peut amener à ceci, personnellement).

Reply:
Fix in a80a8f27fd1a55806ab72635368de4f7dac3f16d